src/components/not-found.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { NimbleEmoji } from '.'
+
+export default class NotFound extends React.PureComponent {
+  constructor(props) {
+    super(props)
+    this.state = { errored: false }
+    console.log(props)
+    this.handleError = this.handleError.bind(this)
+  }
+
+  handleError(event) {
+    this.setState({ errored: true })
+  }
+
+  render() {
+    var { data, emojiProps, i18n, notFound } = this.props
+
+    const imageComponent =
+      !notFound.notFoundImg || this.state.errored ? (
+        NimbleEmoji({
+          data: data,
+          ...emojiProps,
+          size: 38,
+          emoji: 'sleuth_or_spy',
+          onOver: null,
+          onLeave: null,
+          onClick: null,
+        })
+      ) : (
+        // <img src={notFound.notFoundImg} onError={this.handleError} />
+        // <div dangerouslySetInnerHTML={{ __html: notFound.notFoundImg() }} onError={this.handleError} />
```

Comment:
/nit these should also go

Reply:
Ugh, sorry, totally forgot to remove these things